### DISCUSSION TITLE
How to configure issue autocreation by email

### DISCUSSION THREAD

**Discussion by OanaOita** [eventum/eventum#920] [Q&A] [answered]
Where i can find info on How to configure issue auto creation by email.

**Comment by glensc** [comment]
short answer: from wiki: https://github.com/eventum/eventum/wiki

long answer: configure what? why do you even assume you can configure? please share your information first what info you found lacking.

**Comment by OanaOita** [comment]
Ill explain better: I want to configure eventum so as if i send an email to a predefined email address a new issue will be created. If i reply by email to an email received from an issue my email will be attached to the issue as a reply.
Is this possible? I assumed it is possible, reading  some old issues in this forum.

**Comment by glensc** [comment]
https://github.com/eventum/eventum/blob/v3.9.3/docs/wiki/System-Admin/General-Setup.md#email-integration-feature

and what issues? link!

**Comment by OanaOita** [comment]
#201

**Comment by OanaOita** [comment]
I don´t understand how to configure the email integration. Is not a user friendly how to... I think i need to configure something more about email because today i found out that it gives an error if an user clicks on "I forgot my password":

(1/1)                                     ControllerDoesNotReturnResponseException                                                                  The  controller must return a "Symfony\Component\HttpFoundation\Response"  object but it returned null. Did you forget to add a return statement  somewhere in your controller?
--
in ForgotPasswordController.php line 39
in HttpKernel.php line 150
at HttpKernel->handleRaw(object(Request), 2)in HttpKernel.php line 67
at HttpKernel->handle(object(Request), 2)in ControllerTrait.php line 94
at AbstractController->forward('Eventum\\Controller\\ForgotPasswordController::defaultAction', array('request' => object(Request), '_controller' => 'Eventum\\Controller\\ForgotPasswordController::defaultAction'))in FrontController.php line 49
at FrontController->indexAction(object(Request))in HttpKernel.php line 150
at HttpKernel->handleRaw(object(Request), 1)in HttpKernel.php line 67
at HttpKernel->handle(object(Request), 1, true)in Kernel.php line 198
at Kernel->handle(object(Request))in Kernel.php line 48
at Kernel::handleRequest()in forgot_password.php line 18

**Comment by glensc** [comment]
please separate issue for separate bug reports. and do include exact steps to reproduce for your error.

**Comment by OanaOita** [comment]
Ok, I created a new issue for the bug report. Now can you help me with this issue? How can i configure the email routing in version 3.7.1 with php version 7.3.

**Comment by glensc** [comment, marked as answer]
as I already wrote, it's' documented:
- https://github.com/eventum/eventum/blob/v3.9.3/docs/wiki/System-Admin/General-Setup.md
- https://github.com/eventum/eventum/blob/v3.9.3/docs/wiki/System-Admin/General-Setup.md#email-integration-feature

flip the flag and it should work.

or you should be specific what exactly is not working for you and description what have you already done.
